app/assets/stylesheets/ama_layout/layout_components/banners.scss

```diff
@@ -23,6 +23,22 @@
     color: $white;
   }
 
+  &__container{
+    @include grid-row;
+    @include clearfix;
+
+    &--two{
+      @include large-2;
+      @include small-12;
+    }
+
+    &--ten{
+      @include large-10;
+      @include small-12;
+      margin-top: 30px;
```

Comment:
is the point of this margin to center the content in the banner box? if so it might be better to use @include vertical-center so that it's more reusable depending on how much content there is.